.gitignore

```diff
@@ -15,3 +15,6 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
+# Don't check vendor into versioning
+vendor
```

Comment:
We need to get rid of the `vendor` directory completely, so I suggest that you delete the directory instead of suppressing it via `.gitignore`.

Reply:
Done